### DISCUSSION TITLE
Definition of test systems for the model

### DISCUSSION THREAD

**Discussion by datejada** [TulipaEnergy/TulipaEnergyModel.jl#49] [Ideas]
We need to have some test systems for the model. So far, I see these test systems to consider:

1. **Test system using ESDL structure – Decarbonizing the Norse Mythology Energy System**: This is for testing all the modelling possibilities of the model using the ESDL structure.
2. **6-unit test system - SpineOpt**: The MOPO project is developing a 6-unit test system for benchmarking with other tools. We should also consider using that system to compare our model against other tools.
3. **A large-scale EU test system**: This test system could be the more standard type we will solve with the model (as it is now with COMPETES). This test will help to determine the impact of changes in the code on our day-to-day studies with the model.

All these test systems can be added as examples (tutorials?) in the documentation.

Any other ideas/changes?

**Comment by clizbe** [comment]
Maybe also an Industry test system? For a model that is smaller in scale but higher in technology density and interaction?
I think it would be nice if the model an solve this kind of problem to show it's possible, given the right data.

**Comment by clizbe** [comment]
These are good!

**Comment by clizbe** [comment]
Suvayu says:
"Reorganize the tests to make the objectives of these integration tests explicit."

**Comment by clizbe** [comment]
Shall we break these into Issues or is this something for later in the year?

**Comment by datejada** [comment]
We have this covered by the current tests systems, the EU benchmark, and the OBZ case study.
